west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 10da7c808f2c8a30570542d8faa035df9c47816f
+      revision: pull/832/head
```

Comment:
Please update SHA: `814e868e90d0945f9831b73617e4e869d4979739`

Reply:
SHA has been updated and a changelog entry has been added.